bin/etcd-manage

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eo pipefail
+
+preStart() {
+  # pull ip address for current machine and set in environment
+  export ETCD_IP=$(hostname -i)
+  _log "Started etcd at ${ETCD_IP}"
+}
+
+# TODO: Need to stop etcd properly
```

Comment:
I think you want [`./etcdctl member remove <id>`](https://github.com/coreos/etcd/tree/master/etcdctl#member-remove-memberid)